File: Task/Random-numbers/F-Sharp/random-numbers-2.fs
let gaussianRand count =
    let o = new System.Random()
    let pi = System.Math.PI
    let gaussrnd =
        (fun _ -> 1. + 0.5 * sqrt(-2. * log(o.NextDouble())) * cos(2. * pi * o.NextDouble()))
    [ for i in {0 .. (int count)} -> gaussrnd() ]

File: Task/Random-numbers/Standard-ML/random-numbers-2.ml
val seed = (47,42);
val gen = Random.rand seed;
fun random_gaussian () =
  1.0 + Math.sqrt (~2.0 * Math.ln (Random.randReal gen)) * Math.cos (2.0 * Math.pi * Random.randReal gen);
val a = List.tabulate (1000, fn _ => random_gaussian ());
